### DISCUSSION TITLE
Competing risks in DCA

### DISCUSSION THREAD

**Discussion by conan-koh123** [MSKCC-Epi-Bio/decisioncurveanalysis#21] [General]
Hi, I would like to ask about competing risks in DCA. Do we have to generate the competing risk model (eg using stcrreg with STATA) before generating the DCA plot? Currently, the DCA website does not generate the competing risk model, but instead relies on the Cox PH model generated to produce the DCA plot. 

Thank you!

**Comment by VickersA** [comment]
How you generate your model is completely up to you! PH, competing risk, AI, tea leaves, tarot cards: DCA is 100% agnostic. it is a method of evaluating predictions.
